### ISSUE TITLE
Spam Check settings can't be disabled

### ISSUE THREAD

**Issue by jaredatch** [bmarshall511/wordpress-zero-spam#176]
See https://a.cl.ly/Wnubn0Gz

These settings default to enabled but can't be disabled. 

If you uncheck them and save the settings, when the settings page refreshes they are still checked.

**Comment by bmarshall511** [comment]
@jaredatch Thanks for the catch! Just submitted a fix into master (https://github.com/bmarshall511/wordpress-zero-spam/commit/f2badbba7eb303ae60619bf7fabc9265c7876292). If you have a chance, can you test it out before I release? Also, can you shoot me your WP username & I'll add you to the contrib for helping test?

**Comment by jaredatch** [comment]
Yep, I've cloned `master` and pulled latest changes - settings now save as I'd expect :)

wp.org username is also `jaredatch`.

Great work, excited to see the project become active again!

**Comment by bmarshall511** [comment]
Thanks again @jaredatch , I'll get you added in the next release.
